README.md

```diff
@@ -3,18 +3,35 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T1055324)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# FileManager for Web Forms - How to open PDF and Images in popup
-This example demonstrates how to open images and PDF-files using ASPxFileManager, ASPxPanel and ASPxImage controls.
+
+# The file manager for Web Forms - How to open PDF files and images in a popup window
+
+This example demonstrates how to use the [ASPxFileManager](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFileManager), [ASPxPanel](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPanel?p=netframework), and [ASPxImage](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxImage) controls to open **PDF** files and images in a [popup window](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl?p=netframework).
+
 ![Sample](./Sample.png)
```

Comment:
A few notes about the image. 

- It should have borders of some sort. 
- I don't easily see the connection between file thumbnails and the popup window content. I'd rather see an image preview - because it's easy to associate a little image with a big image. 
- The current file displays awards from year 2013 - eight years ago... You won't have this problem with an image preview. 
- The popup control's header section simply says "Header". At least change it to "Preview". The best case is if you can display something like this "Preview: ExportedDocument.pdf". The latter is especially useful if you want to stick with a PDF file preview.

Reply:
I've changed the image and the code. Thank you!